omod/src/main/java/org/openmrs/module/metrics/model/impl/GeneralEndPointConfig.java

```diff
@@ -0,0 +1,38 @@
+package org.openmrs.module.metrics.model.impl;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.ArrayList;
+
+public class GeneralEndPointConfig {
+	
+	private List<GraphiteConfiguration> graphiteConfigurations;
+	
+	public GeneralEndPointConfig() {
+			graphiteConfigurations = new ArrayList<>();
+		}
+	
+	public GeneralEndPointConfig(List<GraphiteConfiguration> graphiteConfigurations) {
+		this.graphiteConfigurations = graphiteConfigurations;
+	}
+	
+	public List<GraphiteConfiguration> getGraphiteConfigurations() {
+		return graphiteConfigurations;
+	}
+	
+	@Override
+	public boolean equals(Object o) {
+		if (this == o)
+			return true;
+		if (o == null || getClass() != o.getClass())
+			return false;
```

Comment:
I realise this is probably auto-generated code, but we really should have curly braces around these `if` statements.

Reply:
Sure yes will add them.